reading/livebook.livemd

```diff
@@ -4,7 +4,8 @@
 Mix.install([
   {:kino, github: "livebook-dev/kino", override: true},
   {:kino_lab, "~> 0.1.0-dev", github: "jonatanklosko/kino_lab"},
-  {:vega_lite, "~> 0.1.3"},
+  {:vega_lite, "~> 0.1.4"},
```

Comment:
I'd like to ensure consistency between notebooks. I'll make a change later today to address it.

Reply:
Noted, I'll not submit future PR with this change.